controller/QtiCreator.php

```diff
@@ -143,42 +150,32 @@ public function saveItem(){
         $this->returnJson($returnValue);
     }
 
-    public function getFile(){
+    public function getFile()
```

Comment:
What happens if the file isn't found ? An exception is thrown ? If yes, IMHO it should be cached and return a 404